### PR TITLE
PAYARA-2201 Document MP Fault Tolerance

### PR DESCRIPTION
**Pull request by Pandrex247**

### CODE REVIEW

File: documentation/microprofile/faulttolerance.adoc [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+|Description
+|Default
+|Mandatory
+
+|`managedexecutorservicename`
+|The JNDI name of the Managed Executor Service to look up.
+|"" (concurrent/__defaultManagedExecutorService)
+|no
+
+|`managedexecutorservicename`
```
Comment (OndrejM) [inline documentation/microprofile/faulttolerance.adoc]: Should be `managedscheduledexecutorservicename`

File: documentation/microprofile/faulttolerance.adoc [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+* Timeout
+
+Since the asynchronous annotation is invoked first, if it's combined with any other fault tolerance annotations they will be processed on the asynchronous thread.
+
+== Asadmin Commands
+
+==== `set-fault-tolerance-configuration`
+
+*Usage*::
+`asadmin> set-fault-tolerance-configuration [--managedexecutorservicename=<managedexecutorservicename>]
```
Comment (OndrejM) [inline documentation/microprofile/faulttolerance.adoc]: Missing ` at the end - it's not correctly formatted as a code snippet

File: documentation/microprofile/faulttolerance.adoc [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/1.0[
+The complete specification] is maintained in the same repository
+
+
+== Additions to the Fault Tolerance Spec
+In addition to the functionality 
+https://github.com/eclipse/microprofile-fault-tolerance/blob/master/spec/src/main/asciidoc/microprofile-fault-tolerance-spec.asciidoc[
+detailed in the spec], the following is also implemented in Payara Server and Payara Micro:
+
+=== Configurable Managed Executor Service and Managed Scheduled Executor Service
```
Comment (OndrejM) [inline documentation/microprofile/faulttolerance.adoc]: The header is too long. I would write only "Configurable Managed Executor Services"

**Comment by mikecroft** [comment]
Just realised my review from before never got submitted >.< Only now that it's merged does it highlight the boxes and show me the comments are still pending! I'll submit it anyway, and we can always handle things in a new PR :+1:

**Pull request merged**